Feat[hospitals]: Hospitals page

medimessarat opened the pull request:
# Responsive simple hospitals View
According to 7d33115. I've created a simple responsive view of hospitals with their picture, title and link to Maps data is stored in data.js file. 

Here is a preview of it: 
![PR_hospitals](https://user-images.githubusercontent.com/40898323/69448981-10e6cb80-0d5a-11ea-873c-f51c5b98cbb3.JPG)




This Fixe the #8,  and branch ready to be merged with master

Commit:
feat[one-hospital]: Adding one hospital prototype

Adding one hospital prototype with location button redirecting to map.

Commit:
feat[hospitals]: Adding multiple hospitals

Adding multiple hospitals in a grid responsive layout.

Commit:
feat[hospitals]: Hospitals page

I've created a simple responsive view of hospitals with their pictures, title andlink to their MAP's location. Data of these hospitals are stores in a data.js file, each hospital is an object that an id, pictureUrl, title, gelocalisation

mohammedi-haroune commented on src/data.js:
```diff
@@ -0,0 +1,115 @@
+    id: uuid(),
+    title:`EPH Bologhine`,
+    imageUrl: `https://cdnfr2.img.sputniknews.com/images//104213/97/1042139753.jpg`,
+    imageUrl: `pictures/hospitals/EPH_Bologhine.jpg`,
```
Why there is two `imageUrl` attributes ?

medimessarat replied:
There is only one imageUrl attributes, but in two ways: 
1.st way that is **Commented**, where we have the link of the picture.
2.nd  way where the picture are saved and we indicate the relative path.

We'll use the second way, but i kept the first thinking that we might need them until we finally deploy our last version. 
The mistake here is that i forgot to comment all the imageUrl that has link redirecting to the picture. that's why we see two imageUrl, but it's solved now in 1633967

mohammedi-haroune commented on package.json:
```diff
@@ -2,10 +2,8 @@
-  "homepage": "http://mohammedi-haroune.github.io/red-crescent-webapp",
   "dependencies": {
     "@material-ui/core": "^4.6.1",
-    "gh-pages": "^2.1.1",
```
why removing gh-pages ?

medimessarat replied:
Sorry my mistake, i didn't **git pull** when i created the hospitals-page branch, so i didn't have the version of package.json where it has gh-pages. 
now it's fixed in 098b66e

mohammedi-haroune commented on package.json:
```diff
@@ -14,9 +12,7 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject",
```
Why removing the deploy scripts ? we need them

medimessarat replied:
it wasn't remove only the comma was removed, from the version of package.json i had. Now it's fixed 098b66e

Commit:
Fix[package.json]: readding the dependecies for pages-deploy-action

mohammedi-haroune commented on package.json:
```diff
@@ -14,9 +14,9 @@
     "predeploy": "npm run build",
-    "deploy": "gh-pages -d build"
+    "deploy": "gh-pages -d build",
+    "eject": "react-scripts eject",
```
There is a syntax error here, it should be
```suggestion
    "eject": "react-scripts eject"
```

Commit:
fix[data]: keeping one imageUrl attribute

i commented all the imageUrl attributes with link to the picture, and
kept only the ones indicating the relative path of the picture.

mohammedi-haroune commented:
I've tested it, I really liked it. 

Just one thing: IMHO, clicking on the location button should open a new that rather than use the current tab.

mohammedi-haroune commented:
A bunch of warning shows up when running:
```
Compiled with warnings.

./src/components/Hospitals/Hospitals.js
  Line 1:17:   'useState' is defined but never used       no-unused-vars
  Line 21:36:  'rest' is assigned a value but never used  no-unused-vars

./src/components/HospitalCard/HospitalCard.js
  Line 1:17:  'useState' is defined but never used     no-unused-vars
  Line 7:3:   'CardActions' is defined but never used  no-unused-vars
  Line 9:3:   'Grid' is defined but never used         no-unused-vars

./src/App.js
  Line 2:8:  'logo' is defined but never used  no-unused-vars
```

mohammedi-haroune commented:
The app name and the app icon should also be fixed

mohammedi-haroune commented:
I have a question, why don't we use the images directly from Wikipedia, this will save us some space and speed up the push/pull process.

Commit:
Fix[package.json]: readding the dependecies for pages-deploy-action

Commit:
Revert "Fix[package.json]: removing comma at 'eject':'react-scripts-eject'

my server won't run if i let a comma at the last attribute of an object
in the package.json, so i removed the comma and now it's working"

This reverts commit ee402549594a4bd614ed0a9134d0ccd4e3b1d991.

Commit:
Revert "Revert "Fix[package.json]: removing comma at 'eject':'react-scripts-eject'"

This reverts commit d092cc2db28e22662f51f668ae22fc73f8d09e10.

Commit:
Fix[package.json]: removing comma at 'eject':'react-scripts-eject'

my server won't run if i let a comma at the last attribute of an object
in the package.json, so i removed the comma and now it's working"

Commit:
fix[MAPs_in_newPage]: Open MAPs in new tab, instead of the same link

Commit:
Fix[Warning]: fixed all the warning, Compiling successfully!

Commit:
Fix[icon_title]: Fixed Icon and Title.

Remove react icon and put redCrescent instead with, Don de Sang(DZ) as
title of the app.

medimessarat commented:
#9 Fixed all reviews, Branch ready to be merged.

mohammedi-haroune commented:
Icon not yet changed

medimessarat commented:
I have changed the **react-app icon**, by the **red-crescent icon** with the same name of the file favicon.ico
Here is how it is in the folder:
![Capture](https://user-images.githubusercontent.com/40898323/69463971-5cf73780-0d7d-11ea-8214-00bce28d0daa.JPG)

mohammedi-haroune commented:
OK. Sorry
